mapviz_plugins/src/pointcloud2_plugin.cpp

```diff
@@ -494,7 +517,7 @@ namespace mapviz_plugins
       case 1:
         return *reinterpret_cast<const int8_t*>(data + feature_info.offset);
       case 2:
-        return *reinterpret_cast<const uint8_t*>(data + feature_info.offset);
+        return *(data + feature_info.offset);
       case 3:
         return *reinterpret_cast<const int16_t*>(data + feature_info.offset);
```

Comment:
`reinterpret_cast<const uint8_t*>` is redundant; `data + feature_info.offset` is a `const uint8_t*`.